### ISSUE TITLE
训练rec_chinese_common_train_v2.0.yml  识别错误。

### ISSUE THREAD

**Issue by yfc886** [PaddlePaddle/PaddleOCR#2413]
训练rec_chinese_common_train_v2.0.yml  这个配置文件的时候，ACC可以达到0.96 LOSS：0.7 左右。 转换成静态图进行识别的话使用训练的图进行识别率测试不到 0.001%     请问大佬这是什么问题？

训练结果：
![111](https://user-images.githubusercontent.com/18112706/113869905-d6ad8b00-97e3-11eb-9e31-999da08de661.jpg)
识别结果：
![222](https://user-images.githubusercontent.com/18112706/113870116-15dbdc00-97e4-11eb-8fcf-0b34c743a498.jpg)

配置文件：

Global:
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_chinese_common_v2.0_shouxie/
  save_epoch_step: 3
  # evaluation is run every 5000 iterations after the 4000th iteration
  eval_batch_step: [0, 2000]
  # if pretrained_model is saved in static mode, load_static_weights must set to True
  cal_metric_during_train: True
  #pretrained_model: ./pretrain_models/ch_ppocr_server_v2.0_rec_pre/best_accuracy
  pretrained_model:
  checkpoints:
  save_inference_dir: ./inference/ch_ppocr_server_v2.0_rec_shouxie
  use_visualdl: False
  infer_img: doc/imgs_words/ch/word_1.jpg
  # for data or label process
  character_dict_path: ppocr/utils/words.txt
  character_type: en
  max_text_length: 15
  infer_mode: False
  use_space_char: True

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0.00004

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: ResNet
    layers: 34
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 256
  Head:
    name: CTCHead
    fc_decay: 0.00004

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/shouxie_train_data/rec/train
    label_file_list: ["./train_data/shouxie_train_data/rec/train/rec_gt.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - RecAug: 
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 32
    drop_last: True
    num_workers: 8

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/shouxie_train_data/rec/test/
    label_file_list:  ["./train_data/shouxie_train_data/rec/test/rec_gt.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 32
    num_workers: 8

**Comment by LDOUBLEV** [comment]
你是怎么转换模型的

**Comment by yfc886** [comment]
@LDOUBLEV  使用  python tools\export_model.py --config=configs/rec/ch_ppocr_v2.0/rec_chinese_common_train_v2.0.yml  这个命令转换模型的。

**Comment by MissPenguin** [comment]
自己训练时换参数和字典了吗
